### PR TITLE
feat(document_loaders): add tencent cos directory and file loader

### PR DESCRIPTION
**Pull request by kagawagao**
- add tencent cos directory and file support for document-loader

#### Who can review?

@eyurtsev

### CODE REVIEW

File: langchain/document_loaders/tencent_cos_directory.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        """Initialize with COS config, bucket and prefix.
+        :param conf(CosConfig): COS config.
+        :param bucket(str): COS bucket.
+        :param prefix(str): prefix.
+        """
+        self.conf = conf
+        self.bucket = bucket
+        self.prefix = prefix
+
+    def load(self) -> List[Document]:
```
Comment (hwchase17) [inline langchain/document_loaders/tencent_cos_directory.py:22]: lets implement lazy_load and then have `load` just call that

**Review by hwchase17 — APPROVED** [review]
can we add an example notebook?

other than that, looks great!

**Comment by kagawagao** [comment]
> can we add an example notebook?
> 
> other than that, looks great!

done

**Review by hwchase17 — APPROVED** [review]
thanks!

**Pull request merged**